config/analytics.yml

```diff
@@ -145,6 +145,18 @@ shared:
     - safeguarding_issue_details
     - training_and_cpds_section_completed
     - imported_steps
+    - following_religion
+    - religious_reference_type
+    - faith
+    - place_of_worship
+    - religious_referee_name
+    - religious_referee_address
+    - religious_referee_role
+    - religious_referee_email
+    - religious_referee_phone
+    - baptism_address
+    - baptism_date
+    - ethos_and_aims
```

Comment:
:raising_hand_man: 
The majority of these new fields could be considered PII. Adding them to `analytics.yml` file will expose them in plain to BigQ Analytics platform. 

This list needs to be curated with our data analyst (Brandon) and:
- Fields containing PII that Brandon wants in analytics will also need to be listed in `analytics_hidden_pii.yml` so they get masked before being pushed to BigQ. 
- Fields containing PII that Brandon doesn't need in BigQ will need to be listed in `analytics_blocklist.yml` **instead** of here.
- Fields not considered PII can be left here.


From the list of new fields here, I would guess the only ones not purely PII are the "following religion" are these three:
```    
- following_religion
- religious_reference_type
- faith
```
They're still personal sensitive data and one would wonder if they actually could be combined with other fields to identify a particular person. 

I guess "following_religion" is generic enough, but I believe the particular faith of an applicant should be masked? 

The docs on the gem referencing the different config files usage: https://github.com/DFE-Digital/dfe-analytics?tab=readme-ov-file#5-send-database-events